### ISSUE TITLE
After pasting image, cannot hit enter (after clicking next to image)

### ISSUE THREAD

**Issue by nevaan9** [froala/wysiwyg-editor#3833]
##### Expected behavior.
I copy and paste an image to the editor, then I CLICK NEXT TO THE IMAGE (to set the cursor on the on right side of the image);
Then I hit enter
The cursor should move to the next line
After I set the cursor next to the image, I should be able to type as well.

##### Actual behavior.
After hitting enter the cursor does not move to the next line
After typing, I cannot type anything

##### Steps to reproduce the problem.
- Copy paste image to the editor
- (The image will be highlighted)
- **CLICK** next to the image (to se the cursor on the right hand side of it)
- Hit enter or type anything

##### Editor version.
Forked the Init/Destroy demo from the froala website demo page
https://jsfiddle.net/nevaan9/9whcjqvr/3/

##### OS.
Mac 10.15.3

##### Browser.
Chrome Version 80.0.3987.100

##### Recording.
froala_issue.mov

**Comment by dejanmartinovic** [comment]
@nevaan9 Can you confirm that the issue can be reproduced in this fiddle?
https://jsfiddle.net/dejan_m/og0hxf2t/3451/

**Comment by nevaan9** [comment]
@dejanmartinovic it seems to be working in that fiddle

**Comment by rogerfar** [comment]
I attached a video where this issue is still happening.

When hitting enter the whole image just moves, instead of going to the next line.

https://streamable.com/auxh6

**Comment by dejanmartinovic** [comment]
@nevaan9 @rogerfar We are aware of that issue. It occurs in the last release (3.1.0). Fix will be available in 3.1.1. Fiddle I've shared running the v3.0.6 and that's why you can reproduce it there.

**Comment by gagadzq** [comment]
I had the same problem with version 3.0.6. The project is developed based on Vue

**Comment by dejanmartinovic** [comment]
@gagadzq Could you double-check the version? If you are using v3.0.6 can you recreate that issue in Sandbox?

**Comment by gabaum10** [comment]
@dejanmartinovic do you have an estimated date for the 3.1.1. release?

**Comment by dejanmartinovic** [comment]
> @dejanmartinovic do you have an estimated date for the 3.1.1. release?

Unfortunately no. Due to unforeseen technical issues, development had to roll the fixes in 3.1.1 into 3.2.0 which will be released this quarter. I'm sorry for not being able to provide more precise info.

**Comment by gabaum10** [comment]
@dejanmartinovic sorry, for the spam, but you mean Q2, correct?  As in by June/July?

**Comment by dejanmartinovic** [comment]
@gabaum10 That is correct. Version 3.2 will be released by the end of June (probably earlier)

**Comment by micheleh** [comment]
@dejanmartinovic we are using V 3.2 and this issue is not yet fixed. What is the timeline? 
This issue risks our project.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
